packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -25,15 +25,63 @@ Customize error pages as part of the sign-in flow.
 
-However, if an error occurs during sign-in, Okta may need to display an error page to the user. To provide a seamless user experience, you can also customize the error page by using the code editor.
+If an error occurs during sign-in, Okta needs to display an error page to the user. You can use the code editor to customize the error page.
 
-> **Note:** A custom error page only appears when an app connects to Okta by using your custom domain. Otherwise, the default Okta error page appears when a critical error occurs or an application is misconfigured. See [Okta deployment models &mdash; redirect vs. embedded](/docs/concepts/redirect-vs-embedded/#redirect-authentication) for more information on Okta-hosted (redirect) functionality.
+> **Note:** A custom error page only appears when an app connects to Okta by using your custom domain. Otherwise, the default Okta error page appears when a critical error occurs, or you haven't properly configured an app. See [Okta deployment models&mdash;redirect vs. embedded](/docs/concepts/redirect-vs-embedded/#redirect-authentication) for more information on redirect functionality.
 
 ### Use the Brands API
 
-The [Brands API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/) allows you to set icons, images, and colors across your Okta-hosted Sign-In Widget, error pages, email templates, and Okta End-User Dashboard all at once. You don't need to set a customized Okta URL domain.
+The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, and colors across the following:
+
+- sign-in page (redirect authentication)
+- error pages
+- email templates
+- Okta End-User Dashboard
+
+You don't need to set a customized Okta URL domain to use the API.
+
+## Content Security Policy (CSP) for your custom domain
+
+Set up a [custom domain](/docs/guides/custom-url-domain/main/) and customize your [CSP](https://content-security-policy.com/) if you also want to customize the sign-in page or error pages. CSP customizations only take effect on custom domains.
+
+To analyze and detect potentially malicious IP addresses that seek to bypass your CSP, use [Okta ThreatInsight](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-threatinsight).
+
+### Known limitations
+
+* Avoid using `meta` tags to customize the CSP. `meta` tags impact the overall policy. It's easier to control CSP customizations by adding trusted origin URLs in the Admin Console. See [Multiple content security policies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#multiple_content_security_policies).
+* If you add too many trusted origins to your custom CSP, the HTTP header size can exceed the limit allowed by some server software. Update the default server limits or reduce the number of trusted origins.
```

Comment:
I would replace "trusted origin URLs" with "Trusted External Resources" because Okta has another feature called Trusted Origins. The CSP customization UI uses the term "trusted external resources"